src/Component/BlogMiddleComp.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
```

Comment:
This component is not really needed. This could have been a function in the parent component or just inline with other tags. Plus name is terrible